libs/brackets-server/.eslintrc

```diff
@@ -50,6 +50,7 @@
 
         "no-trailing-spaces": 0,
         "eol-last": 0,
+        "quotes": ["error", "double", { "avoidEscape": true }]
```

Comment:
What about other brackets code ?
Doesn't this be necessary for embedded-ext only ?

Reply:
We apply this rule to the files following directories :
```
'libs/brackets-server/embedded-ext/**/*.js',
'libs/brackets-server/hacks/**/*.js',
'libs/brackets-server/lib/**/*.js',
```